src/filters/truncate.js

```diff
@@ -2,30 +2,33 @@
 
 angular.module('openshiftCommonUI')
   .filter('truncate', function() {
-    return function(str, charLimit, useWordBoundary, newlineLimit) {
-      if (!str) {
+    return function truncate(str, charLimit, useWordBoundary, newlineLimit, removeExtraChars) {
```

Comment:
Oh I see what you're using it for... Currently the show link is usually in the upper right, so if the line wraps, I'm not sure we've made space. We might have changed that for toasts, though.